content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Schema module
+---
+
+## Overview
+
+`ytt`'s Schema feature provides a way to verify YAML structures and values with the help of annotations.
+Schema currently allows you to provide default data values, and verify that each data value file provided is compatible with the schema.
+
+The schema definition spells out the set of nodes that must make up that kind of YAML document. Documents that are type checked and validated against the schema are guaranteed to be proper.
+
+For example, in the case of Data Values, the schema guarantees to templates that all values exist and are of the correct type. The aleviates templates from doing existence and type checks, themselves.
+
+There are two features to verify a document is compatible with a schema:
+- type checking — comparing the shape of a YAML structure against its schema’s types.
+- validation — determining whether the values contained in a YAML structure are acceptable.
```

Comment:
We're not supporting validation in this release. I think we can nix this paragraph.